app/src/main/java/base/apptrailers/mobile/globant/com/moviesapp/activities/MovieDetailActivity.java

```diff
@@ -25,23 +30,26 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     protected void onStart() {
         super.onStart();
+
+        Bundle bundle = getIntent().getBundleExtra(BUNDLE_EXTRA);
```

Comment:
suggestion: getIntent().putExtra("key",varValue); and retrieve with: getIntent().getExtras(); //retrieve bundle